test/actions/user.test.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * @jest-environment node
+ */
+
+import { addUser } from '../../src/app/actions/user';
+import { user } from '../factories/user';
+
+describe('User:', () => {
+  describe('given valid params', () => {
+    it('creates the user successfully', async () => {
+      const userPayload = { name: user.name, email: user.email, password: user.password };
+      const { data: createdUser } = await addUser(userPayload);
+
+      expect(createdUser.email).toEqual(user.email);
+      expect(createdUser.name).toEqual(user.name);
+    });
+  });
+
+  describe('given email is not unique', () => {
+    it('returns with unprocessable entity error', async () => {
+      const userPayload = { name: user.name, email: 'dev1@nimblehq.co', password: user.password };
```

Comment:
Why pass a specific email if no assertion is set against it?

Reply:
Here, I am testing the logic of failure in case of creating the user with same email. So, in this test I am using the same email address to create the user twice. In the second creation I get the error. I am asserting for the error code here

Reply:
Understood. That makes sense then :-) A minor improvement would be to use naming to give a hint:

```suggestion
      const userPayload = { name: user.name, email: 'unique.user@nimblehq.co', password: user.password };
```

It is a small trick that make tests more readable and maintainable.

Reply:
Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/64/commits/956002a691762209a192135708b0bd8c1c2bbb48